packages/TelegramClient-UI.package/TCUEmojiHelper.class/class/embedEmojisInto..st

```diff
@@ -0,0 +1,17 @@
+utility
+embedEmojisInto: aString
+
+	| reencoded runs |
+	
+	reencoded := (aString convertToEncoding: 'UTF-16') convertFromEncoding: 'UTF-16'.
+	runs := self createUnicodeRunsFrom: reencoded.
```

Comment:
What is a run?

Reply:
runs are **sequences** of characters with the same property, in this case all of the characters have a codepoint greater than 256.

Reply:
Ok, a comment would be great here, too